cloud/google/machineactuator.go

```diff
@@ -186,6 +177,9 @@ func (gce *GCEClient) CreateMachineController(cluster *clusterv1.Cluster, initia
 }
 
 func (gce *GCEClient) Create(cluster *clusterv1.Cluster, machine *clusterv1.Machine) error {
+	if gce.machineSetupConfig == nil {
```

Comment:
Do we need this nil check? There's a check in the gcp-deployer create command to ensure the given filepath isn't empty. There isn't one in gce-machine-controller's main method, but I think maybe the check should be there instead of here, unless there's a reason why we should be able to create a machine controller without the machine setup config.

Reply:
You are correct that it would be simpler and we would not need the nil check if we required the machineSetupConfig to be a non-nil pointer in the NewMachineActuator(...) method. However, there are commands, such as Delete, that do not need the machineSetupConfig and do not have a valid configListPath and would pass in nil.

note that there are pluses and minuses to making it required (simplifies machineactuator.go code) vs making it optional (simplifies usage of machineactuator.go).